db/migrate/20220609141826_create_data_types_table.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# Will be associated with ONT requests, but adding a pipline key to allow
```

Comment:
In this case it seems logical. We are coming from a place of experience so a bit of forward thinking is ok.